source/MaterialXRenderGlsl/GlslMaterial.cpp

```diff
@@ -222,25 +234,42 @@ ImagePtr GlslMaterial::bindImage(const FilePath& filePath, const std::string& un
         int textureLocation = textureHandler->getBoundTextureLocation(image->getResourceId());
         if (textureLocation >= 0)
         {
-            _glProgram->bindUniform(uniformName, Value::createValue(textureLocation), false);
+            getProgram()->bindUniform(uniformName, Value::createValue(textureLocation), false);
             return image;
         }
     }
     return nullptr;
 }
 
+void GlslMaterial::bindUniformOverrides()
+{
+    if (!_override)
+        return;
+
+    // TODO: This is proof-of-concept code, in production this should probably use uniform blocks, or at least precompute the uniform locations.
+    for (int i = 0; i < _override->getPropertyCount(); i++) {
```

Comment:
In this POC we just bind each property as a uniform, as mentioned in comment we should use uniform blocks here.